### PR TITLE
Interaction Menu - Show distance to target on vehicle actions

### PR DESCRIPTION
**Pull request by LinkIsGrim**
**When merged this pull request will:**
- Show distance to vehicle in action name for vehicle actions. See screenshot below.

### IMPORTANT

- If the contribution affects [the documentation](https://github.com/acemod/ACE3/tree/master/docs), please include your changes in this pull request so the documentation will appear on the [website](https://ace3.acemod.org/).
- [Development Guidelines](https://ace3.acemod.org/wiki/development/) are read, understood and applied.
- Title of this PR uses our standard template `Component - Add|Fix|Improve|Change|Make|Remove {changes}`.

### CODE REVIEW

**Comment by johnb432** [comment]
Why not add it when there is an occupant too? With AI occupants it isn't easy to tell which vehicle is which.

File: addons/interact_menu/functions/fnc_createVehiclesActions.sqf [hunk 1]
```diff
@@ -24,7 +24,9 @@ _vehicles apply {
     private _name = getText ((configOf _x) >> "displayName");
     private _ownerName = [_x, true] call EFUNC(common,getName);
     if ("" != _ownerName) then {
-        _name = format ["%1 (%2)", _name, _ownerName];
+        _name = format ["%1 (%2, %3m)", _name, _ownerName, (ACE_player distance _x) toFixed 1];
+    } else {
+        _name = format ["%1 (%2m)", _name, (ACE_player distance _x) toFixed 1];
```
Comment (BrettMayson) [inline addons/interact_menu/functions/fnc_createVehiclesActions.sqf:29]: Since we're doing it in both paths now, imo it should be a variable before the if

**Review by BrettMayson — APPROVED** [review]
Left a comment, but its fine as is too

**Review by johnb432 — APPROVED** [review]
All good for me.

**Pull request merged**